sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items-batch-scoring/image-object-detection-batch-scoring-non-mlflow-model.ipynb

```diff
@@ -0,0 +1,1173 @@
+{
+ "cells": [
```

Comment:
are we keeping both object detection and classification batch scoring notebooks?